ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -410,9 +410,11 @@ private class JdbcLedgerDao(
             conn,
           )
         ) {
-          throw ErrorFactories.offsetOutOfRange(
-            "Pruning offset for all divulged contracts needs to be after the migration offset"
-          )(new DamlContextualizedErrorLogger(logger, loggingContext, None))
+          throw LedgerApiErrors.RequestValidation.OffsetOutOfRange
+            .Reject(
+              "Pruning offset for all divulged contracts needs to be after the migration offset"
+            )(new DamlContextualizedErrorLogger(logger, loggingContext, None))
```

Comment:
I see it was not introduced by your change, but it feel's a bit weird here to create a new instance of the `DamlContextualizedErrorLogger` at spot and with only one single use across the whole file.. something weird is going on here :)